content/api/suppression-list.apib

```diff
@@ -4,48 +4,84 @@ description: Manage your suppression list - a list of recipient email addresses
 
 # Group Suppression List
 
-A suppression list - or exclusion list, as it is sometimes called - stores a recipient's opt-out preferences. It is a list of recipient email addresses to which you do NOT want to send email. The Suppression List API is used to manage your exclusion list entries.  An entry indicates whether the recipient opted out of receiving one of the following:
+A suppression list - or exclusion list, as it is sometimes called - stores a recipient's opt-out preferences.
```

Comment:
"or exclusion list, as it is sometimes called" -> "also known as an exclusion list"